common/app/model/dotcomrendering/DotcomRenderingFrontsModel.scala

```diff
@@ -0,0 +1,72 @@
+package model.dotcomrendering
+
+import common.Edition
+import common.Maps.RichMap
+import common.commercial.EditionCommercialProperties
+import conf.Configuration
+import experiments.ActiveExperiments
+import model.PressedPage
+import navigation.Nav
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.mvc.RequestHeader
+import views.support.{CamelCase, JavaScriptPage}
+
+case class DotcomRenderingFrontsModel(
+    pressedPage: PressedPage,
+    nav: Nav,
+    editionId: String,
+    editionLongForm: String,
+    guardianBaseURL: String,
+    pageId: String,
+    webTitle: String,
+    webURL: String,
+    config: JsObject,
+    commercialProperties: Map[String, EditionCommercialProperties],
+)
+
+object DotcomRenderingFrontsModel {
+  implicit val writes = Json.writes[DotcomRenderingFrontsModel]
+
+  def apply(page: PressedPage, request: RequestHeader, pageType: PageType): DotcomRenderingFrontsModel = {
```

Comment:
Is this the entire request body or just the header? Maybe we could name it requestHeader if the latter?

Reply:
I think you might be right, but calling RequestHeader's 'request' seems to be done across Frontend, so might be worth keeping this this just for consistency of working on the codebase